docs/rules/arazzo/criteria-unique.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: /docs/cli/rules/arazzo/criteria-unique
+---
+
+# criteria-unique
+
+The criteria list must not contain duplicated assertions.
+
+| Arazzo | Compatibility |
+| ------ | ------------- |
+| 1.0.0  | ✅            |
+
+## API design principles
+
+The criteria list must not contain duplicated assertions.
+
+## Configuration
+
+| Option   | Type   | Description                                             |
+| -------- | ------ | ------------------------------------------------------- |
+| severity | string | Possible values: `off`, `warn`, `error`. Default `off`. |
+
+An example configuration:
+
+```yaml
+arazzoRules:
+  criteria-unique: error
+```
+
+## Examples
+
+Given the following configuration:
+
+```yaml
+arazzoRules:
+  criteria-unique: error
+```
+
+Example of an **incorrect** criteria:
+
+```yaml Object example
+successCriteria:
+  - condition: $statusCode == 200
+  - condition: $statusCode == 200
+onSuccess:
+  - name: 'onSuccessActionName'
+    type: 'goto'
+    stepId: 'buy-ticket'
+    criteria:
+      - condition: $response.body.open == true
+      - condition: $response.body.open == true
+onFailure:
+  - name: 'onFailureActionName'
+    type: 'goto'
+    stepId: 'buy-ticket'
+    criteria:
+      - condition: $response.body.open == true
+      - condition: $response.body.open == true
+```
```

Comment:
I'd remove all examples of incorrect criteria - usually users are not reading documentation from top to bottom, or are in a hurry and will copy the wrong example. without noticing.

Reply:
Sure, you can change docs format as you prefer.
Just wanted to keep rules described in the docs.